byoqm/runner/runner.py

```diff
@@ -103,8 +104,8 @@ def _write_to_csv(self, results: Dict):
 
         with open(file_location, "w") as results_file:
             writer = csv.writer(results_file)
-            writer.writerow([f"qualitymodel={self._model_name}"])
-            writer.writerow([f"src_root={self._src_root}"])
+            writer.writerow([f"qualitymodel", self._model_name])
+            writer.writerow([f"src_root", self._shortenPath.__str__()])
```

Comment:
Won't this change the output format to
```
qualitymodel, code_climate
src_root, some/path/here
metric,value
metric1,value1
metric2,value2
...
```

I think its best if we clearly distinguish key-value pairs describing "meta-data" about the run of the program that generated this results from the actual .csv formatted data of the program.

Reply:
As CSV does not have any official way (or best practices) of defining meta data, we can do what we want with it. I do agree that it would be more readable to have a different writing style for the meta data, as that makes it easier to read.

Reply:
I would personally also like the meta-data to be easily distinguishable from the actual data. That is I agree with Caspar, that a different format for the meta-data would be nice so even if I choose to manually read the CSV file, I am not in doubt as to what is meta-data and what is not.

Reply:
CSV does have a format practice for tuple pairs though, which is separated with a ','..

Any suggestions what to do otherwise? I initially used =, but went away from this.